packages/vue/src/components/molecules/SfTile/SfTile.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="sf-tile" :style="style">
+    <!-- @slot Use this slot to replace title -->
+    <slot name="title" v-bind="{ title }">
+      <h3 v-if="title" class="sf-tile__title">
+        {{ title }}
+      </h3>
+    </slot>
+  </div>
+</template>
+<script>
+export default {
+  name: "SfAlert",
+  props: {
+    /**
+     * Tile title
+     */
+    title: {
+      type: String,
+      default: "",
+    },
+    /**
+     * Tile background color
+     */
+    background: {
+      type: String,
+      default: "",
+    },
+    /**
+     * Tile image
+     */
+    image: {
+      type: [String, Object],
+      default: "",
+    },
+  },
+  computed: {
+    style() {
```

Comment:
Can you explain why we need that?

Reply:
it's for readable logic for background img :)